daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/StablePackageTest.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package archive
+
+import java.io.File
+import com.daml.bazeltools.BazelRunfiles
+import com.daml.lf.language.StablePackage
+import org.scalatest._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class StablePackageTest
+    extends AnyWordSpec
+    with Matchers
+    with Inside
+    with BazelRunfiles
+    with Inspectors
+    with TryValues {
+
+  private def resource(path: String): File = {
+    val f = new File(path).getAbsoluteFile
+    require(f.exists, s"File does not exist: $f")
+    f
+  }
+
+  "DA.StablePackages" should {
+
+    lazy val darFile = resource(rlocation("daml-lf/archive/DarReaderTest.dar"))
+    lazy val libPkgs = {
+      val dar = UniversalArchiveDecoder.assertReadFile(darFile)
+      dar.all.view.filter { case (pkgId, _) => pkgId != dar.main._1 }.toMap
```

Comment:
```suggestion
      dar.dependencies.toMap
```


or maybe we need the `view`?

```suggestion
      dar.dependencies.view.toMap
```

Reply:
@remyhaemmerle-da this is still filtering `.all` instead of just using `.dependencies`

Reply:
3364995640fbe7ac75b39215bd2449c128823131